Frontend/src/pages/registration_administration/components/RegistrationAdministrationList.jsx

```diff
@@ -160,17 +179,14 @@ function RegistrationAdministrationTable({
             <Checkbox
               onChange={(_, data) => {
                 if (data.checked) {
-                  setCheckedBoxes(
-                    registrations.map((registration) => {
-                      add(registration.user.id)
-                      return registration.user.id
-                    })
-                  )
+                  registrations.map((registration) => {
```

Comment:
There's nothing to return here now.
```
registrations.forEach((registration) => add(registration.user.id)
```
In fact could replace the whole block to
```
registrations.forEach((registration) => data.checked ? add(registration.user.id) : remove(registration.user.id)
```

Reply:
good point. Changed